qemu/tests/uefi_pkg.py

```diff
@@ -12,20 +12,27 @@ def run(test, params, env):
     1) Check edk2-ovmf package has been installed already.
     2) Qurey file list on edk2-ovmf package.
     3) Make sure the descriptor meta-files will be in file list.
-       2 or 3 files in total.
        For rhel7: 50-ovmf-sb.json, 60-ovmf.json
-       For rhel8 and rhel9: 40-edk2-ovmf-sb.json, 50-edk2-ovmf.json
-       If it supports SEV-ES on rhel8 and rhel9, there are 3 files in total.
+       For rhel8: 40-edk2-ovmf-sb.json, 50-edk2-ovmf.json
+       If it supports SEV-ES, there are 3 files in total.
        40-edk2-ovmf-sb.json, 50-edk2-ovmf.json and 50-edk2-ovmf-cc.json
```

Comment:
I am a little confused, it looks to me that RHEL8 may have only two json files if SEV-ES is not supported, but in cfg, we set number_of_files = 3 for RHEL8

Reply:
yes, the condition is \"the number of JSON files should be less than or equal to the given value\". The key point is that the "filename" elements point to valid files. Thanks.